src/Estimators/tests/RobotDynamicsEstimator/ConfigFolderPath.h.in

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file FolderPath.h(.in)
+ * @authors Stefano Dafarra
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef CONFIG_FOLDERPATH_H_IN
+#define CONFIG_FOLDERPATH_H_IN
+
+#define SOURCE_CONFIG_DIR "@CMAKE_CURRENT_SOURCE_DIR@"
+
+inline std::string getConfigPath()
+{
+    return std::string(SOURCE_CONFIG_DIR) + "/model.ini";
+}
+
+#endif // CONFIG_FOLDERPATH_H_IN
```

Comment:
We have a similar file here:https://github.com/ami-iit/bipedal-locomotion-framework/blob/dd649fe39767141e32baa85060b9198fbf714c43/src/ParametersHandler/tests/ConfigFolderPath.h.in

I wonder if we can manage to avoid duplicates. What do you think @GiulioRomualdi?

Reply:
We may think to have it in the cmake folder. 
I think we can consider it in a separate PR, do you agree?

Reply:
No problem for me